biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
I guess I disagreed with my own comments in the code. :-(

It looks like making it MAJOR for package removal is the most logical and my comment in the code does not seem to make sense. Lets try this patch, I am ok.